### PR TITLE
New Calibration Step and Sector Analysis

### PR DESCRIPTION
**Pull request by iroreilly**
I have added a new calibration step named "hscalibration" and updated messages for missing model output. Also, hagelslag now has the ability to work with sector data by including weights for training the ML model and outputting track data specifically within a given region.

### CODE REVIEW

**Comment by iroreilly** [comment]
I realized some of the folders copied over themselves, so I will delete the old folders and resubmit a new pull request.

File: bin/hsforecast [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/home/tmp/aburke/miniconda3/envs/hail/bin/python
```
Comment (djgagne) [inline bin/hsforecast]: Can you switch this back to /usr/bin/env python? Using your specific path won’t make it usable by others.

**Review by djgagne — CHANGES_REQUESTED** [review]
Can you make sure the paths in the hsdata, hsforecast, and hsplotter use /usr/bin/env python instead of your tmp path? If that isn’t working, we may need to figure out a different path option.

**Comment by iroreilly** [comment]
All of the files in /bin are back to the original python path!

**Review by djgagne — APPROVED** [review]

**Pull request merged**